dtests/src/test/scala/io/snappydata/hydra/TestUtil.scala

```diff
@@ -179,7 +182,8 @@ object TestUtil {
       val logDir = log.listFiles.filter(_.getName.equals("snappyleader.log"))
       if (!logDir.isEmpty) {
         val leaderLogFile: File = logDir.iterator.next()
-        if (leaderLogFile.exists()) dest = dirString + File.separator + ".." + File.separator + ".." + File.separator + dirName
+        if (leaderLogFile.exists())
```

Comment:
Use the SnappyTestUtils.getTempDir() method instead of this one as both are same.